framework/basehandlers.py

```diff
@@ -798,6 +798,10 @@ def FlaskApplication(import_name, routes, pattern_base='', debug=False):
   # Flask apps also have a debug setting that can be used to auto-reload
   # template source code. TODO: investigate using the setting.
 
+
+  # Reject any huge POSTs
+  app.config['MAX_CONTENT_LENGTH'] = 16 * 1024 * 1024  # 16 MB
```

Comment:
This integer and MAX_ATTACHMENT_SIZE should reference the same constant instead of declaring 16MB twice.

Reply:
They are not the same value.  The overall post can be bigger than the max for one attachment.

Reply:
Whoops, you're right! Could you add a comment to MAX_ATTACHMENT_SIZE that MAX_ATTACHMENT_SIZE should always be less than MAX_CONTENT_LENGTH ? It would prevent this case:
- Developer A: Increases MAX_ATTACHMENT_SIZE above MAX_CONTENT_LENGTH. But they don't increase MAX_CONTENT_LENGTH since it's in another file.
- Developer A: Expects a larger file to be accepted but requests get rejected due to the old MAX_CONTENT_LENGTH